deps/media-playback/media-playback/media.c

```diff
@@ -779,7 +998,13 @@ static inline bool mp_media_init_internal(mp_media_t *m,
 		return false;
 	}
 
+	pthread_mutex_lock(&m->mutex);
+
 	m->thread_valid = true;
+	mp_media_init_queue(&m);
```

Comment:
You are initializing the pointer to the pointer, which causes invalid data to changed.  This is specified in the warning. 
This should be:
```C
mp_media_init_queue(m);
```